Patch out unused __gnu_cxx namespace used from stressapptest.
This fixes build issues with libc++.
https://crbug.com/747053
upstream bug: https://github.com/stressapptest/stressapptest/issues/47

diff -Nuar stressapptest-1.0.8/src/sattypes.h stressapptest_new/src/sattypes.h
--- stressapptest-1.0.8/src/sattypes.h
+++ stressapptest_new/src/sattypes.h
@@ -30,7 +30,7 @@
 #include "stressapptest_config_android.h"  // NOLINT
 #else
 #include "stressapptest_config.h"  // NOLINT
-using namespace __gnu_cxx;  //NOLINT
+
 #endif  // __ANDROID__
 using namespace std;
 
